engine/src/main/java/org/camunda/bpm/engine/impl/persistence/entity/JobManager.java

```diff
@@ -213,8 +213,11 @@ public List<AcquirableJobEntity> findNextJobsToExecute(Page page) {
       }
     }
 
-    params.put("jobPriorityMin", engineConfiguration.getJobExecutorPriorityRangeMin());
-    params.put("jobPriorityMax", engineConfiguration.getJobExecutorPriorityRangeMax());
+    boolean jobExecutorAcquireByPriority = engineConfiguration.isJobExecutorAcquireByPriority();
+    long jobExecutorPriorityRangeMin = engineConfiguration.getJobExecutorPriorityRangeMin();
+    long jobExecutorPriorityRangeMax = engineConfiguration.getJobExecutorPriorityRangeMax();
+    params.put("jobPriorityMin", jobExecutorAcquireByPriority && jobExecutorPriorityRangeMin != Long.MIN_VALUE ? jobExecutorPriorityRangeMin : null);
```

Comment:
Why do we set null if the `jobExecutorAcquireByPriority` is configured true but the `jobExecutorPriorityRangeMin` is `Long.MIN_VALUE`?

Reply:
Here we are setting the parameters for [the SQL mapping](https://github.com/camunda/camunda-bpm-platform/blob/4012a96342d4ce97d1946e3ff48bf0dfd74e56e5/engine/src/main/resources/org/camunda/bpm/engine/impl/mapping/entity/Job.xml#L205-L210). If either `jobExecutorAcquireByPriority` is disabled or we are using the default value for the range property we want the query to not contain that parameter.
The alternative would be to pass `jobExecutorAcquireByPriority` to the SQL mapping and only include the range check if enabled. The disadvantage would be that we can either have both or no range properties in the query.

Reply:
Ah I see, it wouldn't make sense to pass min and max since every job priority would satisfy the condition anyway. Thanks!